hydra_agent/querying_mechanism.py

```diff
@@ -4,7 +4,8 @@
 from hydra_agent.hydra_graph import InitialGraph
 import urllib.request
 import json
-from hydrus.hydraspec import doc_maker
+import hydra_python_core
+from hydra_python_core import doc_maker
```

Comment:
Don't need both these imports. You can either import hydra_python_core and use hydra_python_core.doc_maker or import dic_maker directly. I don't see the reason to do both

Reply:
You are right. Already fixed and re-based according to @de-sh commit.